content/tutorials/r_gbif_name_matching/index.Rmd

```diff
@@ -12,70 +12,92 @@ output:
 
 ## Introduction
 
-This tutorial will explain how you can match a list of 'scientific names' to the GBIF taxonomic backbone
+This tutorial will explain how you can match a list of _scientific names_ against the [GBIF backbone taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c).
 
-Important is that you have [rgbif](https://github.com/ropensci/rgbif) and [inborutils](https://inbo.github.io/inborutils/) installed and available:
+Important is that you have the most recent version of [inborutils](https://inbo.github.io/inborutils/) installed and available:
 
-```{r message=FALSE, warning=FALSE}
-library(tidyverse) # tidyverse
-library(rgbif)     # To Match GBIF
-library(inborutils) # wrap GBIF api data
+```{r install_pkgs, eval=FALSE}
+remotes::install_github("inbo/inborutils")   # install inborutils
+```
+
+```{r load_libraries, message=FALSE, warning=FALSE}
+library(tidyverse)    # To do datascience
+library(rgbif)        # To lookup names in the GBIF backbone taxonomy
+library(inborutils)   # To wrap GBIF API data
 library(knitr)
 ```
 
 ## Read data file containing the scientific names
 
 Read file containing the scientific names you want to check against the [GBIF](https://www.gbif.org/species/search?q=) taxonomic backbone:
```

Comment:
Or point to  [GBIF backbone taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c) instead of [GBIF](https://www.gbif.org/species/search?q=)?

Reply:
Via your comment I think actually I found the best option: we can point to this link, which is what we do by using `rgbif` function `name_backbone`:
https://www.gbif.org/species/search?dataset_key=d7dddbf4-2cf0-4f39-9b2a-bb099caae36c&advanced=1
Done here: https://github.com/inbo/tutorials/pull/146/commits/4e050084cbd713fe1dfc1e4c306c8e79faf15e77